build/module-builder/tsconfig.json

```diff
@@ -1,4 +1,19 @@
 {
-  "extends": "../../packages/bp/tsconfig.json",
-  "include": ["src/*.ts"]
-}
+  "compilerOptions": {
+    "strict": true,
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "resolveJsonModule": true,
+    "target": "es2019",
+    "noImplicitAny": false,
+    "moduleResolution": "node",
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "useUnknownInCatchVariables": false,
+    "baseUrl": ".",
+    "outDir": "bin",
+    "types": ["reflect-metadata"]
+  },
+  "include": ["src/**/*"],
+  "exclude": ["**/node_modules/**/*"]
```

Comment:
I think this is the default.

Reply:
I battled hard with those configs to find the sweet spot, I'm not sure I want to touch them anymore haha